cmd/addons_logs.go

```diff
@@ -54,12 +80,20 @@ Allowing you to look at the log output generated by a Home Assistant add-on.
 		if err != nil {
 			fmt.Println(err)
 			ExitWithError = true
-		} else {
-			fmt.Println(string(resp.Body()))
+			return
 		}
+
+		ExitWithError = !helper.StreamTextResponse(resp)
 	},
 }
 
 func init() {
+	addonsLogsCmd.Flags().BoolP("follow", "f", false, "Continuously print new log entries")
+	addonsLogsCmd.Flags().Int32P("lines", "n", 0, "Number of log entries to show")
+	addonsLogsCmd.Flags().StringP("boot", "b", "", "Logs of particular boot ID")
+	addonsLogsCmd.Flags().BoolP("verbose", "v", false, "Return logs in verbose format")
+	addonsLogsCmd.Flags().Lookup("follow").NoOptDefVal = "true"
+	addonsLogsCmd.Flags().Lookup("verbose").NoOptDefVal = "true"
+
```

Comment:
```suggestion

	addonsLogsCmd.RegisterFlagCompletionFunc("follow", boolCompletions)
	addonsLogsCmd.RegisterFlagCompletionFunc("lines", cobra.NoFileCompletions)
	addonsLogsCmd.RegisterFlagCompletionFunc("verbose", boolCompletions)
```
There should also be a `bootCompletions` ideally for the `boot` argument. That way users don't have to separately look up boot ids and copy and paste them. The completion would call `/host/logs/boots` and list out the boot IDs and offsets. See addon completions for an example of how this works:
https://github.com/home-assistant/cli/blob/713510d4333b0bb25d8d0b677d5369bd73861f7c/cmd/addons.go#L46-L101